libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -226,6 +228,14 @@ def with_config(
             bound=self, config={**(config or {}), **kwargs}, kwargs={}
         )
 
+    def with_retry(
```

Comment:
Thoughts on naming it just `retry` ?

Reply:
personal pref for with_retry (retry sounds like it's already failed and is retrying)